src/style.css

```diff
@@ -0,0 +1,136 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
+  margin: 0;
+  width: 200px;
+}
+
+.product {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 2px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.product__img {
+  width: 160px;
+  height: 134px;
+}
+
+.product__name {
+  font-family: inherit;
+  font-size: 11px;
```

Comment:
Check correct font-size

Reply:
If you set correct sizes for card everything will be great with font-size: 12px
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/30314756/161774952-2ac9a0d2-e34b-45e9-88e3-803286f67ea8.png">

Reply:
Font-family: 12px